dir.props

```diff
@@ -151,4 +188,8 @@
     <MakePriExtensionPath>$(_WindowsPhoneKitBinPath)\x86\MrmEnvironmentExtDl.dll</MakePriExtensionPath>
     <MakePriExtensionPath_x64>$(_WindowsPhoneKitBinPath)\x64\MrmEnvironmentExtDl.dll</MakePriExtensionPath_x64>
   </PropertyGroup>
+
+  <Import Project="$(RoslynPropsFile)"
+          Condition="'$(UseRoslynCompiler)'=='true' and Exists('$(RoslynPropsFile)')" />
+
```

Comment:
It appears the way this is setup if the user passes `/p:UseRoslynCompiler` but doesn't install the package it will silently revert to the normal csc.  That could give users the false sense they are compiling with Roslyn when in fact they are using the native compiler.   If you remove the `Exists` portion of the `Condition` it would error in that case instead which seems more appropriate.